api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/CommonCohortQueries.java

```diff
@@ -219,13 +219,19 @@ public CohortDefinition getMohMQPatientsOnCondition(
     if (occurType == "last") {
 
       query +=
-          "SELECT p.person_id FROM person p INNER JOIN "
-              + "(SELECT p.person_id, MAX(e.encounter_datetime) FROM person p INNER JOIN encounter e ";
+          "SELECT p.person_id FROM person p "
```

Comment:
why `select from person` instead of `select from patient`?

Reply:
Because somehow in the same query we need to check patient gender and that info is only available on `person` table!

Reply:
ok thanks  going to merge it